server/dummyData.js

```diff
@@ -26,315 +26,327 @@ module.exports = function() {
     if(!vehicleType) {
-      typeLuxury.name = "luxury";
+      typeLuxury.name = nameCarType3;
       typeLuxury.hourlyRate = 10.5;
-      typeLuxury.save({}).catch(function(err) {
-        if (11000 === err.code || 11001 === err.code) {
-          console.log("DB Error: Duplicate of type:", typeLuxury.name);
-        }
-      });
+      typeLuxury.save({}).catch();
     }
     else {
-      console.log('luxury vehicleType already exists')
-      typeLuxury = vehicleType;
+      console.log("DB Error: vehicleType \'", nameCarType3, "\' already exists.");
+      typeLuxury = vehicleType; //Links variable to existing type
     }
   });
 
 
   let typeSuv = new VehicleType();
-  VehicleType.findOne({'name': 'suv'}, (error, vehicleType) => {
+  var nameCarType4 = "suv";
+  VehicleType.findOne({'name': nameCarType4}, (error, vehicleType) => {
     if(!vehicleType) {
-      typeSuv.name = "suv";
+      typeSuv.name = nameCarType4;
       typeSuv.hourlyRate = 10.5;
-      typeSuv.save({}).catch(function(err) {
-        if (11000 === err.code || 11001 === err.code) {
-          console.log("DB Error: Duplicate of type:", typeSuv.name);
-        }
-      });
+      typeSuv.save({}).catch();
     }
     else {
-      console.log('suv vehicleType already exists')
+      console.log("DB Error: vehicleType \'", nameCarType4, "\' already exists.");
       typeSuv = vehicleType;
     }
   });
 
 
   // add cars if there are none already in the system
-  Car.findOne({'rego': 'ABC123'}, (error, car) => {
+  var regoC1 = 'ABC123';
+  Car.findOne({'rego': regoC1}, (error, car) => {
     if(!car) {
-      let carOne = new Car();
-      (carOne.rego = "ABC123"), (carOne.make = "Hyundai");
-      carOne.model = "Getz";
-      carOne.colour = "white";
-      carOne.year = "2017";
-      carOne.seats = "5";
-      carOne.doors = "3";
-      carOne.vehicleType = typeSmall;
-      carOne.location = locationOne;
-      carOne.movements = new Movement();
-      carOne.movements.car = carOne;
+      let car1 = new Car();
+      (car1.rego = regoC1), (car1.make = "Hyundai");
+      car1.model = "Getz";
+      car1.colour = "white";
+      car1.year = "2017";
+      car1.seats = "5";
+      car1.doors = "3";
+      car1.vehicleType = typeSmall;
+      car1.location = location1;
+      car1.movements = new Movement();
+      car1.movements.car = car1;
       //Movement Block
       let movementsC1 = new Movement();
-      movementsC1.car = carOne;
-      movementsC1.coordinates = locationOne.coordinates;
+      movementsC1.car = car1;
+      movementsC1.coordinates = location1.coordinates;
       movementsC1.save({}).catch();
     
-      carOne.movements = movementsC1;
-      carOne.save({}).catch(function(err) {
-        if (11000 === err.code || 11001 === err.code) {
-          console.log("DB Error: Duplicate of car:", carOne.rego);
-        }
-      });
+      car1.movements = movementsC1;
+      car1.save({}).catch();
```

Comment:
rather than have an empty catch() block can we instead do something like a 

`car1.save().catch((e) => { console.log(e)})`

otherwise we have a silently failing piece of code, and we won't know what is going wrong